### PR TITLE
garflinks IMPROVED???

### PR DESCRIPTION
**Pull request by idiote**
i think i did it without ruining odiekick this time

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -95,13 +95,13 @@
     if message.content.lower() == "odiekick":
         response = "http://66.media.tumblr.com/tumblr_lq5hk354UG1qbqwr6o1_400.gif"
         await message.channel.send(response)
-        
-    if message.content.lower() == "garflinks":
+    
+    if "garflinks" in message.content.lower():
```
Comment (ectocious) [inline bot.py:99]: please keep this line the same as all of the other command if statements before it. the lasagne one is an exception

File: bot.py [hunk 2]
```diff
@@ -111,7 +111,7 @@
             links += "\n \n"
 
         links += "```"
-        await message.channel.send(links)
```
Comment (ectocious) [inline bot.py:114]: you.... took out the thing that sent the message

**Review by ectocious — CHANGES_REQUESTED** [review]

File: bot.py (hunk 2 above)
Comment (idiote) [reply bot.py:114]: im confused

Comment (ectocious) [reply bot.py:114]: `await message.channel.send(links)` is the bit of code that sends the message

**Pull request closed without merging**